src/tests/ftest/launch.py

```diff
@@ -26,6 +26,7 @@
 from argparse import ArgumentParser, RawDescriptionHelpFormatter
 import json
 from multiprocessing import Process
+import sys
```

Comment:
This line should be removed.  There is no new code being added using `sys` and the existing code using `sys` is handled by line 34.